### PR TITLE
Added all tests for Radixsort (incl edge cases)

### PR DESCRIPTION
**Pull request by babyygemperor**
Except for the final sort

**Shortly describe your proposed changes:**
 * None

**Link any issues if they have led to changes in the request:**
 * None

**Note the (sub-)exercises the tests are meant for:**

* GAD 07 - RadixSort

### CODE REVIEW

**Comment by babyygemperor** [comment]
Alright, all tests are here, lemme know when something's off

**Review by N0W0RK — CHANGES_REQUESTED** [review]
Could you create two different PRs for Exercise 6 and 7. I could go ahead and check the changes for 6 but I havent yet had the tim to look at exercise 7

**Comment by babyygemperor** [comment]
Oh shit I didn't notice that my issue fix commit got committed to this branch, lemme try to fix that

**Comment by babyygemperor** [comment]
I'm still waiting for someone to approve the changes to this pull request. It's been a week now

**Comment by CrsiX** [comment]
You didn't eliminate the changes introduced for exercise 6 yet, as outlined by @N0W0RK. After reverting your changes on those files, I guess he will also agree on this PR (and the style checker wouldn't complain about it anymore) :)

I did not add a review on purpose, because I can't guarantee that those tests are correct since I haven't started the exercise yet. The only thing I could say would be "yes, it compiles". That's not meaningful enough to use in a code review :)

File: GAD 7 - Radixsort/tests/radix/FastResult.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
+package tests.radix;
+
+import java.util.ArrayList;
```
Comment (N0W0RK) [inline GAD 7 - Radixsort/tests/radix/FastResult.java:3]: As far as I can tell there should be an import for gad.radix.Result here

File: GAD 7 - Radixsort/tests/radix/RadixsortTester.java [hunk 2]
```diff
@@ -0,0 +1,821 @@
...
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+
+public class RadixsortTester {
```
Comment (N0W0RK) [inline GAD 7 - Radixsort/tests/radix/RadixsortTester.java:14]: The class should be named RadixSortTester

**Review by N0W0RK — CHANGES_REQUESTED** [review]
When you fix the two changes I added and remove the GAD 6 files from this PR I will approve this.

File: GAD 7 - Radixsort/tests/radix/FastResult.java (hunk 1 above)
Comment (babyygemperor) [reply GAD 7 - Radixsort/tests/radix/FastResult.java:4]: Fixed

File: GAD 7 - Radixsort/tests/radix/RadixsortTester.java (hunk 2 above)
Comment (babyygemperor) [reply GAD 7 - Radixsort/tests/radix/RadixsortTester.java]: Renamed

**Comment by babyygemperor** [comment]
I've fixed all the issues and addressed all the code style errors which I could.
I've left the expected results in a bracket, so that it's easier to just collapse the entire hard coded result and easily be able to navigate through the tests. The line length limitation is albeit important, but I don't know how I should address that issue

**Review by N0W0RK — APPROVED** [review]
